File: Iniciante/URI 1013.cs
using System;

class URI
{

  public static void Main(string[] args)
  {

    int MaiorAB = 0;

    string line1 = Console.ReadLine();

    string[] values = line1.Split(' ');
    int a = Int32.Parse(values[0]);
    int b = Int32.Parse(values[1]);
    int c = Int32.Parse(values[2]);

    MaiorAB = (a + b + Math.Abs(a - b)) / 2;

    if (MaiorAB > c)
    {
      Console.WriteLine("{0} eh o maior", MaiorAB);
    }
    else
    {
      Console.WriteLine("{0} eh o maior", c);
    }
  }
}

File: Iniciante/URI 1013.cpp
#include <stdio.h>

int main()
{
  int a, b, c, MaiorAB = 0;

  scanf("%d %d %d", &a, &b, &c);

  if (MaiorAB > c)
  {
    printf("%d eh o maior", MaiorAB);
  }
  else
  {
    printf("%d eh o maior", c);
  }

  return 0;
}